### ISSUE TITLE
Forcing a specific SSID

### ISSUE THREAD

**Issue by vvo** [dominictarr/wifi.sh#11]
How would we implement:

```sh
SSID=specific-SSID wifi.sh connect
```

?

Use case: two internet connections, first one is unavailable due to network problems but still the wifi router give me an IP. I want to use the second wifi with wifi.sh

Searched for "wpa_supplicant select SSID" but it seems it autoselects ("smart" = not so smart here) :D

**Comment by dominictarr** [comment]
yeah, that would be handy. What about creating a temp wpa_supplicant.conf that only contained the desired ssid?

**Comment by vvo** [comment]
Yeah that would work I guess

**Comment by dominictarr** [comment]
I would merge a PR that did this for you if you called `wifi.sh connect <ssid> <pass>`

**Comment by vvo** [comment]
If I can't get no wifi tomorrow I will be forced to do this, wonder if it will be hard or not

**Comment by dominictarr** [comment]
one thing I have learnt from this project (and the reason for it) is that the god given wifi utils in linux are not very user friendly. There is a tonne of edge cases... I could probably give a hour talk about all the things that can go wrong (some I understand, some I just know work arounds)

wpa_supplicant is weird but behaves fairly reliably, but can't connect to open networks. `iw` can connect to open networks, but is weird, and goes into the background... which isn't what you actually want.
Wifi drops out too much... you want to have a terminal open and see the status. It would be good to rewrites those tools into something actually simple... but that could be months of work.

**Comment by dominictarr** [comment]
Shouldn't be too hard if you know a little bash
